github4s/src/test/scala/github4s/integration/GHIssuesSpec.scala

```diff
@@ -273,4 +273,21 @@ trait GHIssuesSpec extends BaseIntegrationSpec {
     response.statusCode shouldBe notFoundStatusCode
   }
 
+  "GHIssues >> GetMilestone" should "return error for an invalid number" taggedAs Integration in {
```

Comment:
could we have a test that is succeeding?

Reply:
We don't have any milestones in this repository.  :(

Reply:
https://github.com/47degrees/github4s/milestone/1 😄